website/api_versioned_docs/version-0.21/zkvm/optimization.md

```diff
@@ -468,17 +468,20 @@ below.
 
 ---
 
-[^1]:
-    Here "sampling" is in quotes because the profiler actually captures the call
+[^1]: Here "sampling" is in quotes because the profiler actually captures the call
     stack at every cycle of program execution. Capturing a call stack on every
     cycle of execution is not done in most programs on physical CPUs for a few
     reasons:
+
     <!-- HACK: This comment prevents the list below from being interpreted to be a code block -->
+
     - It would be cost prohibitive to do so for all but quite short program
       executions.
     - Introducing such heavy profiling would actually alter the performance
       characteristics in significant ways.
+
     <!-- -->
+
```

Comment:
all of these changes will break tests -- basically the `<!-- HACK: -->` comment gets disabled, and this code gets treated as a test block.. Same issue. I spent 5 hours trying to debug this and no matter what, remark keeps formatting footnotes wrong.

Will need to dig a bit deeper.